geometric_kernels/spaces/lie_groups.py

```diff
@@ -0,0 +1,121 @@
+import lab as B
+import json
+from pathlib import Path
+import more_itertools
+import sympy
+from sympy.matrices.determinant import _det as sp_det
+import numpy as np
+from geometric_kernels.lab_extras import dtype_double, from_numpy
+from geometric_kernels.spaces.base import DiscreteSpectrumSpace
+from geometric_kernels.spaces.eigenfunctions import EigenfunctionWithAdditionTheorem, Eigenfunctions
+import abc
+
+
+class LieGroupAddtitionTheorem(EigenfunctionWithAdditionTheorem):
```

Comment:
Naming is weird. The analog for homogenous spaces is called `CompactHomogeneousSpaceEigenfunctions`. It seems that either this class should be "CompactLieGroupEigenfunctions" or `CompactHomogeneousSpaceEigenfunctions` should become `HomogeneousSpaceAdditionTheorem`. Or both should be changed to something else. @stoprightthere what do you think?

Reply:
Not sure the original comment about the naming still stands. Seems like in here and in "homogeneous_spaces.py" the naming is adjusted to a uniform standard?